src/System.Management.Automation/engine/CommandProcessorBase.cs

```diff
@@ -4,9 +4,8 @@
 using System.Collections;
 using System.Collections.ObjectModel;
 using System.Management.Automation.Internal;
-using System.Management.Automation.Language;
 
-using Dbg = System.Management.Automation.Diagnostics;
+#nullable enable
```

Comment:
I can already tell I'm not going to enjoy merging this into my Dispose{} branch for #9900 😂